src/components/access_group/AccessGroupCreate.vue

```diff
@@ -0,0 +1,293 @@
+<template>
+  <main class="group-create">
+    <Button class="button group-create__back-action" @click="handleBackClicked">
+      <Icon id="chevron-left" :width="17" :height="17" />{{
+        fluent('access-group_create', 'group-create__back-action')
+      }}
+    </Button>
+    <section class="primary-area">
+      <h1>{{ fluent('access-group_create') }}</h1>
+      <AccessGroupEditPanel
+        class="details-container"
+        :title="fluent('access-group_details')"
+      >
+        <template v-slot:content>
+          <div class="content-area__row">
+            <label class="content-area__label">{{
+              fluent('access-group_details', 'name')
+            }}</label>
+            <TextInput
+              type="text"
+              v-model="groupName"
+              :maxlength="50"
+              class="content-area__value"
```

Comment:
```suggestion
              class="content-area__value"
              pattern="[a-z_\-]+"
```
group name validation

Reply:
There is not currently any validation supporter of that attribute in the TextInput component